fms_core/resources.py

```diff
@@ -34,10 +44,30 @@ class Meta:
 
 
 class SampleResource(GenericResource):
+    # simple model fields
+    biospecimen_type = Field(attribute='biospecimen_type', column_name='Biospecimen Type')
+    name = Field(attribute='name', column_name='Sample Name')
+    alias = Field(attribute='alias', column_name='Alias')
+    volume = Field(attribute='volume', column_name='Volume (uL)', widget=DecimalWidget())
+    concentration = Field(attribute='concentration', column_name='Conc. (ng/uL)', widget=DecimalWidget())
+    depleted = Field(attribute='depleted', column_name='Source Depleted')
+    experimental_group = Field(attribute='experimental_group', column_name='Experimental Group')
+    collection_site = Field(attribute='collection_site', column_name='Collection Site')
+    tissue_source = Field(attribute='tissue_source', column_name='Tissue Source')
+    reception_date = Field(attribute='reception_date', column_name='Reception Data', widget=DateWidget())
```

Comment:
Reception Data -> Reception Date

Reply:
that's what they have in their template

Reply:
oh ok... we should let them know then.
ill see if i can figure out whats causing the error